pennylane/circuit_graph.py

```diff
@@ -206,6 +211,124 @@ def hash(self):
         """
         return hash(self.serialize())
 
+    def to_openqasm(self, rotations=True):
```

Comment:
curious, what could be a use case of `rotations=False`? it seems to me that the observable used for measurements could be adjusted with the `QNode`, where `qml.PauliZ` could also be specified

Reply:
It's conceivable that a plugin provides its own expectation methods, and if so, we should allow it to apply a circuit without the rotations appended.